.storybook/gu-preview/GuPreview.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-fallthrough */
+import React, { Fragment, ReactElement } from 'react';
+
+import { styled } from '@storybook/theming';
+import { Icons, IconButton } from '@storybook/components';
+import { knobsData } from '../../src/utils/knobsData';
+
+const IconButtonWithLabel = styled(IconButton)(() => ({
+    display: 'inline-flex',
+    alignItems: 'center',
+}));
+
+const IconButtonLabel = styled.div<{}>(({ theme }) => ({
+    fontSize: theme.typography.size.s2 - 1,
+    marginLeft: 10,
+}));
+
+function constructPreviewUrl() {
+    let baseUrl =
+        'https://www.theguardian.com/sport/2019/jul/28/tour-de-france-key-moments-egan-bernal-yellow-jersey?dcr=false&';
+
+    const theKnobs = knobsData.get();
+
+    let fullUrl = baseUrl + 'force-braze-message=' + encodeURIComponent(JSON.stringify(theKnobs));
+    window.open(fullUrl);
+}
+
+export const GuPreview = (): ReactElement => {
+    return (
+        <Fragment>
```

Comment:
I think in general the preferred style at the Guardian is the use the `<>`/`</>` version of fragments (perhaps we should have a rule for this in the eslint config).

Reply:
Good to know, I hadn't encountered that syntax but I'll give it a go.